k2/csrc/array_ops_test.cu

```diff
@@ -9,21 +9,24 @@
  * See LICENSE for clarification regarding multiple authors
  */
 
-#include <cstdio>
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
+
+#include <cstdio>
 #include <numeric>
 #include <random>
 #include <utility>
 #include <vector>
 
 #include "k2/csrc/array.h"
-#include "k2/csrc/context.h"
-#include "k2/csrc/ops.h"
+#include "k2/csrc/array_ops.h"
+#include "k2/csrc/context.cuh"
 #include "k2/csrc/timer.h"
 
 namespace k2 {
 
+#define int int32_t
```

Comment:
No!!!!

Reply:
oh, this is test code, as I found the Transpose() error: about int (not int32_t).. It's still not fixed, I commented two Transpose() lines there to get compiled

Reply:
fixed